java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -4342,37 +4342,78 @@ void testMakeStruct() {
 
   @Test
   void testMakeListEmpty() {
-    final int numRows = 10;
-    try (ColumnVector expected =
+
+    final int numRows = 4;
+    List<List<Integer>> mainList1 = new ArrayList<>();
+    mainList1.add(Arrays.asList());
+    List<List<Integer>> mainList2 = new ArrayList<>();
```

Comment:
mainList1-4 are all the same, so we could simplify this to something like:
```
List<List<String>> emptyListOfList = new ArrayList<>(new ArrayList<String>());
```
and just reference `emptyListOfList` multiple times.

Reply:
Seems it's not gonna work, according to the code, there will no any element in emptyListOfList
``` java
    public ArrayList(Collection<? extends E> c) {
        Object[] a = c.toArray();
        if ((size = a.length) != 0) {
            if (c.getClass() == ArrayList.class) {
                elementData = a;
            } else {
                elementData = Arrays.copyOf(a, size, Object[].class);
            }
        } else {
            // replace with empty array.
            elementData = EMPTY_ELEMENTDATA;
        }
    }
```